static/scripts/core.js

```diff
@@ -653,3 +661,13 @@ window.addEventListener('hashchange', (event) => {
         bringIdToViewOnMount(url.hash);
     }
 });
+
+// eslint-disable-next-line no-undef
+window.addEventListener('storage', event => {
+    if (event.newValue === 'undefined') return;
+
+    if (event.key === themeLocalStorageKey)
+        localUpdateTheme(event.newValue);
+    if (event.key === fontSizeLocalStorageKey)
+        localUpdateFontSize(event.newValue);
```

Comment:
@javalsai It is working great, but font-size value is not in synced in the UI. It is because we are not updating the tooltip.

Please add `initTooltip` function at the end to update the tooltip value when there is any change in localStroage.

Reply:
Ok, I won't have time to do so until tomorrow, but I'll add this and make a test.js file as you said in the other PR.

Reply:
Should be fixed now